GetJobProfiles/CypherCommandRecipes/CreateOccupationLabelNodes.recipe.json

```diff
@@ -12,7 +12,7 @@
         {
             "name": "CypherCommand",
             "commands": [
-                "match (o:esco__Occupation)\r\n[token:whereClause]\r\nunwind o.skos__altLabel as altLabels\r\ncreate (al:OccupationLabel:Resource {skos__prefLabel: altLabels, uri: \"<<contentapiprefix>>/occupationlabel/\" + apoc.create.uuid()})\r\ncreate (o)-[:hasAltLabel]->(al)"
+                "with 4 as hexLength\r\nmatch (o:esco__Occupation)\r\n[token:whereClause]\r\nunwind o.skos__altLabel as altLabels\r\nwith o, altLabels, hexLength, '0x' + right( o.uri, hexLength) as baseHexCode\r\nwith o, altLabels, hexLength, baseHexCode, toLower(apoc.text.hexValue( apoc.convert.toInteger(baseHexCode) + apoc.coll.indexOf(o.skos__altLabel,altLabels) + 1)) as replacementChars, left(right( o.uri, 36),36 - hexLength) as trimmedUri, 0 as zero\r\nwith o, altLabels, hexLength, replacementChars, baseHexCode, trimmedUri + case size(replacementChars) when hexLength then replacementChars else 1/zero end as newUri\r\ncreate (al:OccupationLabel:Resource {skos__prefLabel: altLabels, uri: \"<<contentapiprefix>>/occupationlabel/\" + newUri})\r\ncreate (o)-[:hasAltLabel]->(al)"
```

Comment:
Looks like the 1/zero is a mechanism to raise an error?
We could instead use `right(replacementChars, hexlength)` to cope with overflow without raising an error.